installer/conf/omsagent.d/baseline.conf

```diff
@@ -0,0 +1,30 @@
+# Baseline plugins
+
+<source> 
+    type exec
+    tag oms.baseline_immediate_run
+    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d 

```

Comment:
What is a correct way to measure CPU and Memory? 
The applications runs about 20 seconds and take less that 10% CPU (from top command)